LICENSE

```diff
@@ -6,7 +6,10 @@ PyMC is distributed under the Apache License, Version 2.0
 
 Copyright (c) 2006 Christopher J. Fonnesbeck (Academic Free License)
 Copyright (c) 2007-2008 Christopher J. Fonnesbeck, Anand Prabhakar Patil, David Huard (Academic Free License)
-Copyright (c) 2009-2017 The PyMC developers (see contributors to pymc-devs on GitHub)
+Copyright (c) 2009-2022 The PyMC developers (see contributors to pymc-devs on GitHub)
```

Comment:
while we are editing this, this copyright sentence should not have end date

Reply:
Should it be `Copyright (c) 2009 The PyMC developers (see contributors to pymc-devs on GitHub)`? or something else?

Reply:
I like the `2009-present` version most

Reply:
@ricardoV94 @oriolabril, let's fix the dates later, because then "while we're at it" we can update it in all places

for all intends and purposes of this PR it's ready to merge now that the attribution part has been taken care of